build/Targets/BeforeCommonTargets.targets

```diff
@@ -103,17 +103,19 @@
   </Choose>
 
   <!--
-    If TargetNetFX20 is true the project targets Framework 2.0 reference assemblies provided by Microsoft.NetFX20 nuget package.
+    If TargetNetFX20 is true the project targets Framework 2.0 reference assemblies provided by Microsoft.NetFramework.ReferenceAssemblies nuget package.
```

Comment:
"TargetNetFX20" still mentioned in the comment...

Reply:
Fixed.